msteams-platform/TOC.yml

```diff
@@ -363,6 +366,8 @@
     href: https://docs.microsoft.com/graph/toolkit/get-started/build-a-microsoft-teams-tab
   - name: Enable resource-specific consent
     href: graph-api/rsc/resource-specific-consent.md
+  - name: Bot APIs with RSC permissions
```

Comment:
"Bots with RSC permissions"

Reply:
This is an error from my end, I should have removed this entry as it is duplicated in two places. As recommended by you keeping it under bots/how-to/conversations/channel-messages-with-rsc.md. Removing this duplicate entry.